tests/cypress/e2e/basicE2E.cy.ts

```diff
@@ -15,14 +15,15 @@ describe('landingBsx', () => {
   })
 })
 
-describe.skip('snekCollectionItem', () => {
+describe('snekCollectionItem', () => {
   it('collectionActions', { scrollBehavior: false }, () => {
     cy.visit('/snek/gallery/659233203-5')
-    cy.snekCollectionActions('3132385849', 'Morski pes', 'bXkmyH...J5CSGP')
+    cy.visit('/snek/collection/3132385849')
+    cy.checkCollectionActions('Morski pes', 'bXkmyH...J5CSGP')
```

Comment:
I would prefer to put '/snek/collection/3132385849' as a param of `checkCollectionActions` function. In this way, we could visit the related page inside the function.